src/main/java/org/spongepowered/common/registry/type/item/EquipmentTypeRegistryModule.java

```diff
@@ -60,12 +62,17 @@ public void registerDefaults() {
         this.equipmentTypeMap.put("any", new SpongeEquipmentType("any"));
         this.equipmentTypeMap.put("equipped", new SpongeEquipmentType("equipped"));
         // TODO - evaluate whether the id's are used for any other purpose than the catalog id type to look up
-        final SpongeEquipmentTypeWorn head = new SpongeEquipmentTypeWorn("head", 0);
+        final SpongeWornEquipmentType head = new SpongeWornEquipmentType("head", EntityEquipmentSlot.HEAD);
         this.equipmentTypeMap.put("headwear", head);
         this.equipmentTypeMap.put("head", head);
-        this.equipmentTypeMap.put("chestplate", new SpongeEquipmentTypeWorn("chestplate", 1));
-        this.equipmentTypeMap.put("leggings", new SpongeEquipmentTypeWorn("leggings", 2));
-        this.equipmentTypeMap.put("boots", new SpongeEquipmentTypeWorn("boots", 3));
-        this.equipmentTypeMap.put("worn", new SpongeEquipmentTypeWorn("worn", 0));
+        this.equipmentTypeMap.put("chestplate", new SpongeWornEquipmentType("chestplate", EntityEquipmentSlot.CHEST));
```

Comment:
At this point, why not make these registrations method calls instead of repeating the same field access over and over again with the same strings being used?